fileio/ft_read_headshape.m

```diff
@@ -1341,7 +1342,84 @@
     shape.hex = shape.hex(:,1:8);
     shape.hex = shape.hex + 1; % this should be one-offset
 
-    
+  case {'neurojson_jmesh' 'neurojson_bmesh'}
+    if(fileformat == 'neurojson_bmesh')
+        shape.orig = loadbj(filename);
```

Comment:
The original `orig` field should not be stored in the output structure. It exposes file-format specific details, which (from experience) result in headaches in the long long-term when our own or others' code starts using it. 

The fieldtrip mesh data structure itself should be as small as possible but as complete/extended as necessary. We can later always add something to make it more complete, but we cannot remove something (as that breaks the API compatibility).

Reply:
`shape.orig` is dropped